app/1.0/shell.html

```diff
@@ -0,0 +1,24 @@
+<!--
+@license
+Copyright (c) 2016 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+<!DOCTYPE html>
+<html itemscope itemtype="http://schema.org/Organization" lang="en">
+<head>
+<title>{% block title %}{% endblock %} - Polymer Project</title>
+{% include 'templates/head-meta.html' %}
+</head>
+<body class="fullbleed" unresolved>
+  <pw-shell>
+    {% include 'templates/site-nav.html' %}
```

Comment:
Since every page load replaces the contents of `<pw-shell>`, it shouldn't need to have any children in the app shell file. (site nav and devguide-shell).